### PR TITLE
Add tesseract build for foxy

### PR DESCRIPTION
**Pull request by Minipada**
I'm adding a build for Foxy for tesseract.

1. I also changed the build arguments since there is no `foxy-perception` tag yet. This follows the work started on https://github.com/ros-industrial-consortium/trajopt_ros/pull/194
2. rviz has been renamed rviz2 on foxy
3. octomap not available on foxy

@mpowelson, you probably want to review this.

### CODE REVIEW

**Comment by mpowelson** [comment]
I'll tag @Levi-Armstrong as well since he was the author of these images.

File: tesseract/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ARG ROSDISTRO=melodic
+ARG ROS_BASE_TAG=melodic-perception
 
-FROM ros:$ROSDISTRO-perception
+FROM ros:$ROS_BASE_TAG
 MAINTAINER Levi Armstrong
 
 ARG ROSDISTRO
```
Comment (Levi-Armstrong) [inline tesseract/Dockerfile:7]: Missing `ARG ROS_BASE_TAG` here.

**Review by Levi-Armstrong — CHANGES_REQUESTED** [review]

File: tesseract/Dockerfile (hunk 1 above)
Comment (Minipada) [reply tesseract/Dockerfile:7]: It's already at line 2

**Comment by Levi-Armstrong** [comment]
@Minipada Also the dockers are not building in CI.

**Comment by Levi-Armstrong** [comment]
It looks like you cannot pass two args to --build-arg. So need to change the line below removing the --build-arg and change CI to have `--build-arg <ARG1> --build-arg <ARG2>`

https://github.com/ros-industrial/docker/blob/8bee1905322346adc422b0c626ba326efe047b6d/build.sh#L25

**Review by Levi-Armstrong — APPROVED** [review]

**Pull request merged**